xcell/mappers/mapper_ACT_base.py

```diff
@@ -8,6 +9,10 @@ def __init__(self, config):
 
     def _get_ACT_defaults(self, config):
         self._get_defaults(config)
+        if self.coords != 'C':
+            self.rot = hp.Rotator(coord=['C', self.coords])
+        else:
+            self.rot = None
```

Comment:
I think we need to do one more thing here: to make sure the "lite" maps are saved consistently, we should add the coordinates to the file names (both for the map and the mask)

Reply:
This would be done in the `get_signal_map` and `get_mask` methods, but I can't add comments there